sdk/samples/iot/paho_iot_pnp_component_sample.c

```diff
@@ -0,0 +1,626 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+/*
+ * This sample connects an Azure IoT Plug and Play enabled device.
+ *
+ * Azure IoT Plug and Play requires the device to advertise its capabilities in a device model. This
+ * sample's model is available in
+ * https://github.com/Azure/opendigitaltwins-dtdl/blob/master/DTDL/v2/samples/TemperatureController.json.
```

Comment:
I could technically get away with this, because the model repo can't really change URLs or there'll be issue to server side applications which take dependencies on these URLs.  

But that's technicality, and I do prefer single source of truth, so changing per suggestion.